feature/policy_forwarding/otg_tests/mpls_gre_udp_decap_test/README.md

```diff
@@ -0,0 +1,158 @@
+# PF-1.7 Decapsulate MPLS in GRE and UDP

```

Comment:
@dplore `PF`? not `TE`? (same applies to this whole doc)

Reply:
The paths used here are only part of the `/policy-forwarding` tree, so PF sounds good.